config/routes.rb

```diff
@@ -31,6 +31,10 @@
   get "/oidc/done", to: "oidc#done"
   get "/oidc/kubeconfig", to: "oidc#kubeconfig"
 
+  resource :orchestration, only: [] do
+    match "/retry-bootstrap", action: :retry_bootstrap, via: [:put, :patch]
```

Comment:
As I said in my comment, change it to a mere create action (also notice that with a `put` an ID was expected, and you are not doing this here either...)

Reply:
It's not a very RESTy approach. You can allow `put` or `patch` without the need of an ID if what you are updating is a (let's call it) singleton resource, like for example a global `/config` endpoint. It makes sense to allow `put` and `patch` on the global `/config` resource without the need of an id.